### DISCUSSION TITLE
File Not Found Error

### DISCUSSION THREAD

**Discussion by marcusbriggsdiss** [PMEAL/OpenPNM#2711] [Q&A]
When I try and load sandstone data (slice images), it says: FileNotFoundError: [Errno 2] No such file or directory: 'sandstone images/sandstone imagessandstone images/S1 5micron AVG4_2031.tif'. 

The line of code I use to try import this file is: raw_file = np.fromfile('sandstone images/'+name+'sandstone images/S1 5micron AVG4_2031.tif', dtype=np.uint8). 

Does anyone know how to fix this problem?

**Comment by jgostick** [comment]
Due to the increasingly high volume of questions that we receive between PoreSpy and OpenPNM, our small team has been finding it very difficult to answer everyone. For this reason we must decline to do basic "tech-support" for problems which are general python/numpy/programming issues. For these type of questions, you should be able to find the answers you need using a search engine and/or StackOverflow. 

I hope you appreciate that we are not trying to be rude, but just trying to prevent burnout and protect our time to focus on our research activities, including developing new features for PoreSpy and OpenPNM. 

If you find that our documentation is lacking, then we would be very happy to receive an example in the form of a Jupyter Notebook that outlines your solution for future users.  We have an `examples/contrib` folder in both PoreSpy and OpenPNM repos which are intended to house such contributions.  You can either email it to us, or use the pull request functionality of GitHub.

> Note: This long response was 'auto-generated' using the [saved replies](https://docs.github.com/en/get-started/writing-on-github/working-with-saved-replies/using-saved-replies) feature.
